app/presenters/candidate_interface/personal_details_review_presenter.rb

```diff
@@ -63,7 +63,7 @@ def english_main_language_row
         key: I18n.t('application_form.personal_details.english_main_language.label'),
         value: @languages_form.english_main_language&.titleize,
         action: ('if English is your main language' if @editable),
-        change_path: candidate_interface_languages_path,
+        change_path: candidate_interface_edit_languages_path,
       }
     end
 
```

Comment:
I think that the change path on `other_language_details_row` needs to be updated too 👇

Reply:
@george-bit thanks, changed in 8285b75. Although I confess that I have no idea when/how these are used...!